README.md

```diff
@@ -172,6 +173,26 @@ export const apiKey = 'SOMEKEY'
 export const API_KEY = 'SOMEKEY'
 ```
 
+- Use `//` for single line comments and `/* */` for multi-line comments.
+
+```javascript
+// bad
+
+// This is a very long comment that continues on and goes over multiple lines, 
+// saying something very long and detailed which all developers will appreciate 
+// reading and thinking about.
+
+/* This is just a single line comment. */
+
+// good
+
+/* This is a very long comment that continues on and goes over multiple lines, 
+ * saying something very long and detailed which all developers will appreciate 
+ * reading and thinking about. */
```

Comment:
**question**: Do we want the `*/` and `/*` on their own lines? For example:
```javascript
/* 
 * This is a very long comment that continues on and goes over multiple lines, 
 * saying something very long and detailed which all developers will appreciate 
 * reading and thinking about.
 */
```

Looking at other style guides for reference, it looks like both Google and Airbnb have similar rules.

I think I prefer having the extra newlines, as it gives the block a bit more room. Interested if anybody else has thoughts though.

At the very least, I think we should allow both options as valid.

https://github.com/airbnb/javascript#comments
https://google.github.io/styleguide/jsguide.html#formatting-comments

Reply:
I personally prefer having `*/` on it's own line for the same reasons as you Caleb, it gives more space to the comment. Happy with either option though.

Reply:
Oh thanks for noticing this @calebissharp, my preference is for 

```
/*
 * A multi-line
 * comment
 */
```

I find this easier to add a new line to the bottom of.

Reply:
I've always looked at those as "typed-style comments" -- I typically use them for C / C++ / etc but find them kind of ugly in JS. I love code with good visual hierarchy 😆  

That said, it looks like eslint has an actual rule for those, but neither of them are the style that I've adopted:

These ones are similar to @calebissharp's:

```
/*
 * this line
 * calls foo()
 */
```

and these ones are similar to my style, but without 

```
/* this line
   calls foo() */
```

Since it sounds like there is a trend of preferring the former I will:
1. Remove the note on comments in this PR 
2. make a separate PR to apply the eslint rule 

which is in-line with our guidelines on _Don't include rules in this guide that could be covered by eslint or prettier configurations_

Thanks for pointing out!

Reply:
Created https://github.com/TwoStoryRobot/eslint-config/pull/27. Will merge this now. Thanks again!